hnn_core/tests/test_viz.py

```diff
@@ -161,5 +167,22 @@ def test_dipole_visualization():
         net.cell_response.plot_spikes_hist(trial_idx='blah')
     net.cell_response.plot_spikes_hist(trial_idx=0, show=False)
     net.cell_response.plot_spikes_hist(trial_idx=[0, 1], show=False)
+    net.cell_response.plot_spikes_hist(color='r')
+    net.cell_response.plot_spikes_hist(color=['C0', 'C1'])
+    net.cell_response.plot_spikes_hist(color={'beta_prox': 'r',
+                                              'beta_dist': 'g'})
```

Comment:
Do we have a test that checks that it works fine when there are two drives defined to belong to the same grouping in `spike_types` ?

Reply:
Added!